src/Computer.php

```diff
@@ -217,11 +218,12 @@ public function post_updateItem($history = 1)
                      $item->getFromDB($tID);
                     if (!$item->getField('is_global')) {
                         $changes['id'] = $item->getField('id');
+                        $item_input = $changes;
```

Comment:
No strictly related to this PR, but could be enhanced.
```suggestion
                        $item_input = $changes;
                        $item_input['id'] = $item->getID();
```

Reply:
done !